config/routes_webhook.php

```diff
@@ -4,11 +4,17 @@
 
 namespace Symfony\Component\Routing\Loader\Configurator;
 
+use CommerceWeavers\SyliusTpayPlugin\Controller\InitApplePayPaymentAction;
 use CommerceWeavers\SyliusTpayPlugin\Controller\PaymentNotificationAction;
 use CommerceWeavers\SyliusTpayPlugin\Routing;
 use Symfony\Component\HttpFoundation\Request;
 
 return function(RoutingConfigurator $routes): void {
+    $routes->add(Routing::INIT_APPLE_PAY_PAYMENT, Routing::INIT_APPLE_PAY_PAYMENT_PATH)
+        ->controller(InitApplePayPaymentAction::class)
+        ->methods([Request::METHOD_POST])
+    ;
```

Comment:
In theory it fits to the `routes_shop` and it's not a webhook (at least in a common sense), but I really wanted to get rid of the locale from the URL, and it was the simplest way without creating a separate URL.